### DISCUSSION TITLE
support automatic screenshot

### DISCUSSION THREAD

**Discussion by M09Ic** [projectdiscovery/katana#331] [Ideas]
Many times I have a list of URLs, but unable to distinguish whether they are valuable.  This is when automatic screenshots and manual judgments can speed up the work.

**Comment by tarunKoyalwar** [comment]
@M09Ic  we already have this option implemented in nuclei and can be customised using template . template id is `screenshot` https://nuclei.projectdiscovery.io/templating-guide/protocols/headless/#screenshot

**Comment by M09Ic** [comment]
Maybe crawlers and screenshots are the best combination. nuclei seems to be used to simulate browser actions, not to take screenshots of each url.

**Reply by ehsandeep** [reply]
@M09Ic taking screenshots of each crawled page without context is expensive; what's the use-case you looking for this feature?

**Comment by MetzinAround** [comment]
Closing this for now, but feel free to reopen if you'd like @M09Ic
